pwiz_tools/Skyline/Util/UiModes.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+ * Original author: Nicholas Shulman <nicksh .at. u.washington.edu>,
+ *                  MacCoss Lab, Department of Genome Sciences, UW
+ *
+ * Copyright 2019 University of Washington - Seattle, WA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using pwiz.Common.Collections;
+using pwiz.Common.DataBinding;
+using pwiz.Skyline.Model;
+using pwiz.Skyline.Properties;
+
+namespace pwiz.Skyline.Util
+{
+    /// <summary>
+    /// Information about the UI modes that the user can switch between in Skyline.
+    /// </summary>
+    public static class UiModes
+    {
+        public const string PROTEOMIC = "proteomic";
+        public const string SMALL_MOLECULES = "small_molecules";
+        public const string MIXED = "mixed";
+
+        public static readonly IEnumerable<string> ALL =
+            ImmutableList.ValueOf(new[] {PROTEOMIC, SMALL_MOLECULES, MIXED});
+
+        public static string FromDocumentType(SrmDocument.DOCUMENT_TYPE documentType)
+        {
+            switch (documentType)
+            {
```

Comment:
Does this take into account DOCUMENT_TYPE.none (ie empty document?)

Reply:
The mode that gets passed in is nearly always SkylineWindow.ModeUI, which I believe would never be "none". If it were "none", then the user would see the Mixed UI mode.

Reply:
Correct, there's never a "none" UI mode. But for that "nearly always" part it's probably better to default to Proteomic to avoid frightening the majority of our users who are traditionally proteomic.